app/views/spree/admin/products/_primary_taxon_form.html.haml

```diff
@@ -2,5 +2,5 @@
   = f.label :primary_taxon_id, t('.product_category')
   %span.required *
   %br
-  = f.collection_select(:primary_taxon_id, Spree::Taxon.all, :id, :name, {:include_blank => true}, {:class => "select2 fullwidth"})
+  = f.collection_select(:primary_taxon_id, Spree::Taxon.all.sort_by(&:name), :id, :name, {:include_blank => true}, {:class => "select2 fullwidth"})
```

Comment:
It's better to let the database sort instead of rails:

```suggestion
  = f.collection_select(:primary_taxon_id, Spree::Taxon.order(:name), :id, :name, {:include_blank => true}, {:class => "select2 fullwidth"})
```

Maybe you still need the `all` to convert it into an array. I'm not sure.

Reply:
nice, it works!